packages/app/src/cli/models/extensions/schemas.ts

```diff
@@ -78,4 +72,5 @@ export const BaseFunctionMetadataSchema = schema.define.object({
   ),
 })
 
-export type NewExtensionPointType = schema.define.infer<typeof NewExtensionPointsSchema>
+export type NewExtensionPointSchemaType = schema.define.infer<typeof NewExtensionPointSchema>
+export type NewExtensionPointsSchemaType = schema.define.infer<typeof NewExtensionPointsSchema>
```

Comment:
Do we need to expose both types here? the second one is just an array of the first one, so if we need it we can just use `NewExtensionPointSchemaType[]` right?

Reply:
Yeah, fair.  I've removed `NewExtensionPointsSchemaType` and it's a little easier to understand now, thanks 👍